source/validate.h

```diff
@@ -36,33 +46,267 @@
 typedef struct spv_id_info_t {
   uint32_t id;
   SpvOp opcode;
-  const spv_instruction_t* inst;
-  spv_position_t position;
+  std::vector<uint32_t> words;
```

Comment:
What does the field mean?
Actually, I'd like a blurb for each member. 
What is opcode?